### ISSUE TITLE
More action items before competition finishes

### ISSUE THREAD

**Issue by minfawang** [minfawang/cs231n-fashion#4]
- [ ] Model ensemble
- [ ] Heuristics: limit max number of output labels in testing
- [ ] Check overfitting in threshold analysis
- [ ] Fine-tune smaller model
- [ ] Feature extractor ensemble
